Documentation/triage/issues.md

```diff
@@ -0,0 +1,46 @@
+# Issue Triage Guidelines
+
+## Purpose
+
+Speed up issue management.
+
+The `etcd` issues are listed at https://github.com/etcd-io/etcd/issues
+and are identified with labels. For example, an issue that is identified
+as a bug will eventually be set to label `area/bug `. New issues will
+start out without any labels, but typically `etcd` maintainers and active contributors
+add labels based on their findings. The detailed list of labels can be found at
+https://github.com/kubernetes/kubernetes/labels
+
+Following are few predetermined searches on issues for convenience:
+* [Bugs](https://github.com/etcd-io/etcd/labels/area%2Fbug)
+* [Help Wanted](https://github.com/etcd-io/etcd/labels/Help%20Wanted)
+* [Longest untriaged issues](https://github.com/etcd-io/etcd/issues?utf8=%E2%9C%93&q=is%3Aopen+sort%3Aupdated-asc+)
+
+## Scope
+
+These guidelines serves as a primary document for triaging an incoming issues in
+`etcd`. Everyone is welcome to help manage issues and PRs but the work and responsibilities discussed in this document are created with `etcd` maintainers and active contributors in mind.
+
+## Validate if an issue is a bug
+
+Validate if the issue is indeed a bug. If not, add a comment with findings and close trivial issue. For non-trivial issue, wait to hear back from issue reporter and see if there is any objection. If issue reporter does not reply in 30 days, close the issue.
```

Comment:
Some issues are reported with only error messages but with no reproduce steps, which makes it difficult to validate if it is indeed a bug. Maybe add something like "if more information is needed from issue reporter, ..."?

Reply:
@jingyih sure, that's a good idea, will do!